luminance/src/shader/program.rs

```diff
@@ -810,12 +822,18 @@ impl fmt::Display for Type {
       Type::ISampler1D => f.write_str("isampler1D"),
       Type::ISampler2D => f.write_str("isampler2D"),
       Type::ISampler3D => f.write_str("isampler3D"),
+      Type::ISampler1DArray => f.write_str("isampler1DArray"),
+      Type::ISampler2DArray => f.write_str("isampler2DArray"),
       Type::UISampler1D => f.write_str("uSampler1D"),
       Type::UISampler2D => f.write_str("uSampler2D"),
       Type::UISampler3D => f.write_str("uSampler3D"),
+      Type::UISampler1DArray => f.write_str("uSampler1DArray"),
```

Comment:
May you please do as the way it’s done everywhere else? `usampler1DArray`.

Reply:
Just one line above it’s `uSampler3D`, I just copied how it is done.  Good catch, those were invalid. :)